### ISSUE TITLE
data(vax,can): COVID-19 Vaccines Currently Authorized by Health Canada

### ISSUE THREAD

**Issue by aywi** [owid/covid-19-data#2618]
### Country

Canada

### Domain

Vaccinations

### Which data is inaccurate or missing?

According to <https://covid-vaccine.canada.ca/> and <https://www.canada.ca/en/health-canada/services/drugs-health-products/covid19-industry/drugs-vaccines-treatments/vaccines.html>, the following list of COVID-19 vaccines are currently being authorized by Health Canada:

|      COVID-19 Vaccine       |           WHO Name            |      OWID Name       | Date of Approval |
| :-------------------------: | :---------------------------: | :------------------: | :--------------: |
|  Pfizer-BioNTech Comirnaty  | `Pfizer BioNTech - Comirnaty` |  `Pfizer/BioNTech`   |  **2020-12-09**  |
|      Moderna Spikevax       |     `Moderna - Spikevax`      |      `Moderna`       |  **2020-12-23**  |
|    AstraZeneca Vaxzevria    |   `AstraZeneca - Vaxzevria`   | `Oxford/AstraZeneca` |  **2021-02-26**  |
| Janssen (Johnson & Johnson) |   `Janssen - Ad26.COV 2-S`    |  `Johnson&Johnson`   |  **2021-03-05**  |
|      Novavax Nuvaxovid      |      `Novavax-NUVAXOVID`      |    **`Novavax`**     |  **2022-02-17**  |
|      Medicago Covifenz      |               /               |  (**`Medicago`**?)   |  **2022-02-24**  |

### Why do you think the data is inaccurate or missing?

1. Since all the above COVID-19 vaccines are currently being authorized under the *Food and Drugs Regulations*, which equals full authorization in many other countries, it is necessary to add all of them to the list of COVID-19 vaccines used in Canada.
2. Medicago Covifenz COVID-19 vaccine doesn't have the corresponding WHO name up to now, and it could be called `Medicago` like other vaccines.

**Comment by lucasrodes** [comment]
Hi @aywi,
Thanks for your help once again.

Currently, both Novavax and Medicago are not listed in our source (https://covid19tracker.ca/vaccinationtracker.html).

Perhaps @noahlittle has some insights?

**Comment by aywi** [comment]
> Hi @aywi, Thanks for your help once again.
> 
> Currently, both Novavax and Medicago are not listed in our source (https://covid19tracker.ca/vaccinationtracker.html).
> 
> Perhaps @noahlittle has some insights?

Yes. But it did mention the latest update is on 2022-02-03, way before the approval of the Novavax and Medicago. Since Novavax has [production site in Canada](https://www.cbc.ca/news/canada/montreal/montreal-novavax-vaccine-production-facility-1.6233282) and the government [started to receive shipments](https://toronto.citynews.ca/2022/03/31/novavax-vaccine-canada-availibility/) one months ago, Nuvaxovid is currently [available in various provinces](https://www.canada.ca/en/public-health/services/diseases/2019-novel-coronavirus-infection/prevention-risks/covid-19-vaccine-treatment/vaccine-rollout.html). Medicago is a Canadian company and Covifenz [are expected to be available in late May](https://globalnews.ca/news/8770404/quebec-mrna-covid-19-vaccines-preferable-to-medicago/).

**Comment by aywi** [comment]
@lucasrodes @noahlittle Latest update from [health-infobase.canada.ca](https://health-infobase.canada.ca/covid-19/vaccine-administration/):
1. Novavax has been administered since 2021-04-24. I have updated the above table (with other corrections to the vaccination timeline).
2. Medicago has not been administered up to now.

**Comment by aywi** [comment]
@lucasrodes Another question: Since the [manufacturer](https://health-infobase.canada.ca/covid-19/vaccine-administration/) and the [age group](https://health-infobase.canada.ca/covid-19/vaccination-coverage/) data are available from Public Health Agency of Canada, is it acceptable to add them as individual sources to the current batch script? The only problem is that their total_vaccinations don't match with the current source, but it looks acceptable for some ECDC countries.
